src/tools/pub/package-layout.md

```diff
@@ -85,26 +85,25 @@ Every package has a [_pubspec_](/tools/pub/pubspec), a file named
 `pubspec.yaml`, in the root directory of the package. That's what *makes* it a
 package.
 
-Once you've run [`pub get`](/tools/pub/cmd/pub-get),
+Running [`pub get`](/tools/pub/cmd/pub-get),
 [`pub upgrade`](/tools/pub/cmd/pub-upgrade), or
-[`pub downgrade`](/tools/pub/cmd/pub-downgrade) on the package, you will also have a
+[`pub downgrade`](/tools/pub/cmd/pub-downgrade) on the package creates a
 **lockfile**, named `pubspec.lock`. If your package is an [application
 package](/tools/pub/glossary#application-package), check the lockfile into source
 control. Otherwise, don't.
 
+For more information, see the [pubspec page](/tools/pub/pubspec).
+
+## LICENSE
+
 {% prettify none %}
 enchilada/
-  .packages
+  LICENSE
 {% endprettify %}
 
-Running pub also generates a `.packages` file.
-Don't check this into source control.
-
-The open source community has a few other files that commonly appear at
-the top level of a project: `LICENSE`, `AUTHORS`, etc. If you use any
-of those, they can go in the top level of the package too.
-
-For more information, see the [pubspec page](/tools/pub/pubspec).
+If you're publishing your package, you need a license file.
+The [Pub site]({{site.pub}}) accepts files named `LICENSE`,
+`COPYING`, or `UNLICENSE`, with any file extension. 
```

Comment:
I would prefer if we only documented `LICENSE`, I'm not convinced we should encourage users to use different names for license files...

Maybe in some future we want to formalize the packaging format, and then I could imagine that we would require it be called `LICENSE`. It's unfortunate that we accept multiple filenames right now, as it leaves room for ambiguity.

Reply:
Side note: If you want to make a recommendation, you can recommend using an OSI approved license as this allows other to reuse your work.
Maybe even point out that all packages by the Dart team is released under "BSD-3-Clause" (https://spdx.org/licenses/BSD-3-Clause.html).